### ISSUE TITLE
Evaluate method does not pass arguments for metric

### ISSUE THREAD

**Issue by VIGNESHinZONE** [deepchem/deepchem#2295]
After training a model, I wanted to see the evaluation metric for the model. I wanted to calculate the root mean square and this is calculated using 
```
dc.metrics.mean_squared_error( squared=False)
```

I had a problem in this piece of code given  - 
```
metric = dc.metrics.Metric(dc.metrics.mean_squared_error)
print('training set score:', model.evaluate(train, [metric]))
```

In the above piece of code, Root mean square cannot be found as there is no argument being passed to set  squared=False.
I look at the evaluate method given [here](https://github.com/deepchem/deepchem/blob/968e521dafba059f64d97014d36fd1e1ede7f5f1/deepchem/models/models.py#L158) and this does not pass kwargs argument to change the parameters of the dc.metrics. I was using Torch_Models which had inherited a base class called "Model".

**Comment by rbharath** [comment]
Hmm, good point! We source `dc.metrics.mean_squared_error` from https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html which has kwargs but we don't provide for a way to set them ourselves. One workaround is you can use `dc.metrics.rms_score` which computes the root-mean-squared error out of box. 

We should possibly think about enabling passage of keyword arguments to the underlying sklearn metrics though
